src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -227,28 +227,31 @@ const MigrateWalletCompletionComponent = (props: Props) => {
 
   const renderNextButton = React.useMemo(() => {
     return (
-      <MainButton
-        spinner={!done}
-        disabled={!done}
-        label={!done ? undefined : lstrings.string_done_cap}
-        type="secondary"
-        marginRem={[1]}
-        onPress={() => navigation.navigate('walletsTab', { screen: 'walletList' })}
-        alignSelf="center"
-      />
+      <View style={styles.bottomButton}>
```

Comment:
Rant: This is what Jon has been harping on - we should be using the `ButtonsView` to put a button on a scene in a particular layout, instead of centering it ourselves.

*However*, I see that this is a special case, since we are floating the button at the bottom of the screen. Since we are in a hurry, there is no need to go adding a special mode to the `ButtonsView`! Arguably this could even go in the scene footer area, but again, no need to do that in this PR.

Reply:
Ah. Good to know.

I also was inclined to revisit this and other scenes with floating components to instead render these components int he footer.